common/src/main/java/com/microsoft/identity/common/internal/logging/ILoggerCallback.java

```diff
@@ -25,7 +25,10 @@
 /**
  * Interface for apps to configure the external logging and implement the callback to designate the
  * output of the log messages.
+ *
+ * This class is deprecated. Use {@link com.microsoft.identity.common.logging.ILoggerCallback}.
  */
+@Deprecated
 public interface ILoggerCallback {
```

Comment:
nit: also extends the new ILoggerCallback?

Reply:
That would be a breaking change (since the new method from the parent class would be pushed onto the interface)